src/routines/widgets.ycp

```diff
@@ -243,29 +243,44 @@ map<string, any> mtu_widget()
     ];
 }
 
-define string GetDeviceDescription(string device_id) {
-    string device_name = NetworkInterfaces::GetValue(device_id,"NAME");
-    if ( (device_name == nil) || (device_name == "") ) {
-        //TRANSLATORS: Informs that device name is not known
-        device_name = _("Unknown device");
-    }
+// Provides human readable device description
+// @param device_id system wide device name (e.g. eth0)
+// @return hwname, or if it is empty, a translated generic name based on devtype
```

Comment:
Sorry, I meant this wording to be applied to `CheckEmptyName`, where it is really needed because of the confusing name.